lua/null-ls/utils/init.lua

```diff
@@ -353,4 +353,23 @@ M.cosmiconfig = function(...)
     return require("null-ls.utils.cosmiconfig")(...)
 end
 
+--- Get vcs root
+---@return string|nil
+M.get_vcs_root = function()
+    local uv = vim.uv or vim.loop
+    local cwd = uv.cwd()
+    ---@diagnostic disable-next-line: param-type-mismatch
+    local vcs_root = M.root_pattern(".git", ".hg", ".svn", ".bzr")(cwd)
+    if vcs_root then
+        return vcs_root
+    end
+    --- git worktree dir doesn't contains `.git` folder.
```

Comment:
worktree checkouts have a `.git` file pointing to the bare repo. not sure if this is really needed

Reply:
Just double checked, you are right, I think this is not needed.

Reply:
Removed the system call.